dojo/api_v2/views.py

```diff
@@ -1059,28 +1509,33 @@ def get_duplicate_cluster(self, request, pk):
         request_body=no_body,
         responses={status.HTTP_204_NO_CONTENT: ""},
     )
-    @action(detail=True, methods=['post'], url_path=r'duplicate/reset')
+    @action(detail=True, methods=["post"], url_path=r"duplicate/reset")
     def reset_finding_duplicate_status(self, request, pk):
-        finding = self.get_object()
-        checked_duplicate_id = reset_finding_duplicate_status_internal(request.user, pk)
+        self.get_object()
+        checked_duplicate_id = reset_finding_duplicate_status_internal(
+            request.user, pk
+        )
         if checked_duplicate_id is None:
             return Response(status=status.HTTP_400_BAD_REQUEST)
         return Response(status=status.HTTP_204_NO_CONTENT)
 
     @extend_schema(
         request=OpenApiTypes.NONE,
         parameters=[
-            OpenApiParameter("new_fid", OpenApiTypes.INT, OpenApiParameter.PATH)
+            OpenApiParameter(
+                "new_fid", OpenApiTypes.INT, OpenApiParameter.PATH
+            )
         ],
         responses={status.HTTP_204_NO_CONTENT: ""},
     )
     @swagger_auto_schema(
-        responses={status.HTTP_204_NO_CONTENT: ""},
-        request_body=no_body
+        responses={status.HTTP_204_NO_CONTENT: ""}, request_body=no_body
+    )
+    @action(
+        detail=True, methods=["post"], url_path=r"original/(?P<new_fid>\d+)"
     )
-    @action(detail=True, methods=['post'], url_path=r'original/(?P<new_fid>\d+)')
     def set_finding_as_original(self, request, pk, new_fid):
-        finding = self.get_object()
+        self.get_object()
```

Comment:
Is there a good reason to call this without using it?

Reply:
Sorry, changes apply this commit https://github.com/DefectDojo/django-DefectDojo/pull/8321/commits/48d3037e3c39c1db66a2ca449728c2991ec19fc2